### ISSUE TITLE
Create a Spanish version of README

### ISSUE THREAD

**Issue by rapejim** [rapejim/pms-plexdrive-docker#10]
Create a README.ES.md file with the same information but in spanish.

**Comment by nimendrak** [comment]
hi @rapejim, I am working on this issue.

**Comment by rapejim** [comment]
> hi @rapejim, I am working on this issue.

Ok, assigned! 😜

P.D.: Are you spanish, or spanish speaker?

**Comment by nimendrak** [comment]
I am not Spanish but I've studied Spanish back in 2014.

**Comment by jossemarGT** [comment]
Oh chocks, this issue is already taken (:sobs in spanish:). Jokes aside, feel free to add me on the PR as reviewer if you need an extra pair of eyes, I am from LATAM.

**Comment by rapejim** [comment]
> Oh chocks, this issue is already taken (:sobs in spanish:). Jokes aside, feel free to add me on the PR as reviewer if you need an extra pair of eyes, I am from LATAM.

Thanks for your offer. We can do the following, I can merge the current PR but not close the issue, and you @jossemarGT can make a new PR with improvements in the translation (now some parts are very literal).
Ok? 😅

**Comment by jossemarGT** [comment]
Oh my, I didn't mean to slow you down. But sure I can gladly take it as soon you merge the PR, I see #11 is still open .

**Comment by rapejim** [comment]
> Oh my, I didn't mean to slow you down. But sure I can gladly take it as soon you merge the PR, I see #11 is still open .

Merged! 😜

**Comment by jossemarGT** [comment]
If you are still there @rapejim, what is the tone we are looking for this README?

I have to ask since in Spanish we use "you" differntly depending the context; "tu" is a friendly way of speaking and "usted"  is used for a more formal/neutral approach. Also some verb conjugations change a little based on the tone as well.

For the record, I will go with the friendly one. We can change it once the PR is ready.

**Comment by rapejim** [comment]
> If you are still there @rapejim, what is the tone we are looking for this README? I have to ask since in Spanish we use "you" differntly depending the context; "tu" is a friendly way of speaking and "usted"  is used for a more formal/neutral approach.
> 
> For the record I will go with the friendly one, we can change it once the PR is ready.

I know it, (I am Spanish, 🙈🙈 but don't have time to translate it jejeje).
Mejor usa el "tú" o la entonación informal, 😜

**Comment by jossemarGT** [comment]
Ha! I should have figured with the `Europe/Madrid` timezone in the command examples. Well it is done, please take a look at #12

**Comment by rapejim** [comment]
> Ha! I should have figured with the `Europe/Madrid` timezone in the command examples. Well it is done, please take a look at #12

I have taken a look at the PR and I have left you some suggestions.
